internal/internal_workflow_testsuite.go

```diff
@@ -1047,9 +1047,9 @@ func (env *testWorkflowEnvironmentImpl) ExecuteActivity(parameters ExecuteActivi
 	return activityInfo
 }
 
-// Copy of the server function func (v *decisionAttrValidator) validateActivityScheduleAttributes
+// Copy of the server function func (v *commandAttrValidator) validateActivityScheduleAttributes
```

Comment:
Add TODO somewhere to remove these functions. Let's rely on the server-side validation only.